packages/fxa-settings/src/components/PageDeleteAccount/index.tsx

```diff
@@ -9,7 +9,7 @@ import { useAccount, useAlertBar } from '../../models';
 import InputPassword from '../InputPassword';
 import FlowContainer from '../FlowContainer';
 import VerifiedSessionGuard from '../VerifiedSessionGuard';
-import { HomePath, MonitorLink, ROOTPATH, VPNLink } from '../../constants';
+import { HomePath, ROOTPATH } from '../../constants';
```

Comment:
It looks like both `MonitorLink` and `VPNLink` are not longer used. We could remove them from constants file.

Reply:
Removed!